api/v1beta1/cluster_types_test.go

```diff
@@ -183,6 +183,26 @@ func TestClusterIPFamily(t *testing.T) {
 			expectErr: "services: too many CIDRs specified",
 			c:         clusterWithNetwork(nil, []string{"192.168.0.0/16", "fd00:100:96::/48", "10.128.0.0/12"}),
 		},
+		{
```

Comment:
Do we need test cases which test the good cases? I guess single stack IPv4, single stack IPv6 (possible?), dualstack consistently configured for pods and services?

Reply:
I'll add these.